File: solutions/035.search-insert-position/search-insert-position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        int length = nums.length,result=0;
        if(length<=0){
            return 0;
        }
        for(int i = 0;i<length;i++){
            if(nums[i]>=target){
                result = i;
                break;
            }
            if(i == length-1){
                result = length;
            }
        }
        return result;
    }
}

File: solutions/035.search-insert-position/search-insert-position.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert = function(nums, target) {
    let res = -1;
    for(let i = 0; i < nums.length; i++){
        if(nums[i] >= target){
            res = i;
            break;
        }
    }
    if(res === -1) res = nums.length;
    return res;
};